WeatherService.Tests/ParsersTesting/ConfigServiceTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Moq;
+using WeatherService.Models;
+using WeatherService.Services.FileReaders;
+using FluentAssertions;
+
+namespace WeatherService.Tests.ParsersTesting;
+
+public class ConfigServiceTests
+{
+    [Fact]
+    public void LoadConfigSettings_ShouldReturnDeserializedDictionary()
```

Comment:
```suggestion
    public void LoadConfigSettings_ValidJsonInput_ShouldReturnDeserializedDictionary()
```

Also, let's consider the sad scenario.